src/Kunstmaan/UtilitiesBundle/Command/CipherCommand.php

```diff
@@ -20,14 +25,31 @@ class CipherCommand extends ContainerAwareCommand
         3 => 'Decrypt file'
     ];
 
+    public function __construct($cipher = null)
```

Comment:
Same remarks as previous PR, add docblocks everywhere.

Reply:
And change null comparisons with null coalesce operator

Reply:
Docblocks and typehints are fixed. The null coalesce change I didn't do yet because of comment in the other PR. https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/1788#discussion_r162570690